G_channels.md

```diff
@@ -5,20 +5,22 @@ Since Elixir is based on message passing, you may wonder why we need this extra
 The word "Channel" is really shorthand for a layered system with a number of components. Let's take a quick look at them now so we can see the big picture a little better.
 
 #### The Moving Parts
+
+- Socket Handlers
+
+Phoenix holds a single connection to the server and mutiplexes your channel sockets over that one connection. Socket handlers, such as your `web/channels/user_socket.ex`, are modules that authenticate and identify a socket connection and allow you to set default socket assigns for use in all channels.
+
 - Channel Routes
 
-These are defined in the `router.ex` file as all other routes are. They match on the topic string and dispatch matching requests to the given Channel module. The star character `*` acts as a wildcard matcher, so in the following example route, requests for `sample_topic:pizza` and `sample_topic:oranges` would both be dispatched to the `SampleTopicChannel`.
+These are defined in Socket handlers, such as your `web/channels/user_socket.ex` file as all other routes are. They match on the topic string and dispatch matching requests to the given Channel module. The star character `*` acts as a wildcard matcher, so in the following example route, requests for `sample_topic:pizza` and `sample_topic:oranges` would both be dispatched to the `SampleTopicChannel`.
```

Comment:
I think we need to get rid of the "as all other routes are." part here. Perhaps we can add something like this:

. . . file, which makes them distinct from other routes.